spot_rl_experiments/spot_rl/real_policy.py

```diff
@@ -43,8 +44,21 @@ def __init__(
     ):
         print("Loading policy...")
         self.device = torch.device(device)
-        self.is_torchscript_policy = "torchscript" in checkpoint_path
+        config = construct_config()
+        self.is_torchscript_policy = "torchscript" in str(config.WEIGHTS_TYPE).lower()
```

Comment:
Maybe pass config from Nav/Pick/Place Policy classes in call to `RealPolicy` class to avoid presence of 2 different copies of config.